internal/consensus/reactor.go

```diff
@@ -687,6 +691,20 @@ OUTER_LOOP:
 				"height", prs.Height,
 				"vote", vote,
 			)
+		} else {
+			if c := sendEntireCommit(logger, conR.conS, rs, ps, prs, rng); c != nil {
```

Comment:
Can we rename `sendEntireCommit` to `getEntireCommitToSend` or something like that?